docs/articles/vs-test-adapter/Tips-And-Tricks.md

```diff
@@ -350,10 +351,17 @@ run.*"
 Exceptions outside test cases are reported with its stack trace included in the message.  One example here is exceptions
 in OneTimeSetUp and OneTimeTearDown. They belong to the outer suite, and their exceptions are reported as part of the
 message. This option includes the stack trace in that message.  If this becomes too much information, this option turns
-the stack trace reporting off.
+the stack trace reporting off.  Default is true.
 
 (From version 4.0.0)
 
+#### IncludeStackTrace
+
+This is default true.  If turned off (false) it will stop outputting any stacktraces.  This is in cases where you just
```

Comment:
Would like to modify the language here.

Reply:
Please do :-)

Reply:
@OsirisTerje didn't realize at the time that this had been merged without review, so I'll have to check it another time.